roboflow/util/folderparser.py

```diff
@@ -194,6 +194,13 @@ def _filterIndividualAnnotations(image, annotation, format, imgRefMap, annotatio
             return _annotation
         else:
             return None
+    elif format == "multilabel_csv":
+        rows = [r for r in parsed["rows"] if r["file_name"] == image["name"]]
+        if rows:
+            labels = rows[0]["labels"]
+            return {"type": "classification_multilabel", "labels": labels}
+        else:
+            return None
```

Comment:
Avoid iterating all data if you need only first item.

```suggestion
        rows = [r for r in parsed["rows"] if r["file_name"] == image["name"]]
        row = next(r for r in parsed["rows"] if r["file_name"] == image["name"], None)
        if row:
            labels = row["labels"]
            return {"type": "classification_multilabel", "labels": labels}
        else:
            return None
```